deeppavlov/core/layers/bert.py

```diff
@@ -0,0 +1,233 @@
+                                                      layer_norm_epsilon=layer_norm_epsilon,
+                                                      trainable=trainable,
+                                                      name=f'layer_{i}'))
+        # The "pooler" converts the encoded sequence tensor of shape [batch_size, seq_length, hidden_size] to a tensor
+        # of shape [batch_size, hidden_size]. This is necessary for segment-level (or segment-pair-level) classification
+        # tasks where we need a fixed dimensional representation of the segment.
+        # We "pool" the model by simply taking the hidden state corresponding to the first token. We assume that this
+        # has been pre-trained
+        self.pooler = tf.keras.layers.Dense(pooler_fc_size, activation='tanh', trainable=trainable, name='pooler/dense')
+
+    def call(self,
+             inputs: tf.Tensor,
+             training: Optional[bool] = None,
+             mask: Optional[bool] = None,
+             **kwargs) -> tf.Tensor:
+
+        emb = self.embed(inputs, training=training)
+        emb_norm_do = self.embed_dropout(self.embed_layer_norm(emb), training=training)
+
+        # always compute mask if it is not provided
+        if mask is None:
+            mask = tf.cast(tf.not_equal(inputs, self.pad_token_index), tf.int32)
+        enc = self.encoder(emb_norm_do, training=training, mask=mask)
+        # For classification tasks, the first vector (corresponding to [CLS]) is used as the "sentence vector". Note
+        # that this only makes sense because the entire model is fine-tuned.
+        po = self.pooler(tf.squeeze(enc[:, 0:1, :], axis=1))
+
+        if self.return_stack is None:
+            return po
+        elif self.return_stack:
+            raise NotImplementedError('Currently all encoder layers output could not be obtained. You can get '
+                                      'sequence output from the last encoder layer setting return_stack to False')
+        else:
+            return enc
+
+
+class TransformerBlock(tf.keras.layers.Layer):
+    """
+    One block of transformer architecture.
+
+    Args:
+        hidden_size: hidden size of the Transformer (d_model)
+        intermediate_size: the size of the intermediate dense layer
+        num_heads: number of attention heads in the Transformer
+        hidden_dropout_prob: dropout probability for the hidden layers
+        attention_probs_dropout_prob: dropout probability of the attention probabilities
+        intermediate_act_fn: the non-linear activation function to apply to the output of the intermediate dense layer
+        layer_norm_epsilon: some small number to avoid zero division during layer normalization
+        trainable: whether the layer's variables should be trainable
+        **kwargs: keyword arguments for base Layer class
+    """
+    def __init__(self,
+                 hidden_size: int = 768,
```

Comment:
Remove default values for hidden_size, intermediate_size, num_heads and hidden_dropout_prob.